### PR TITLE
Todo List Structure

### PR DESCRIPTION
**Pull request by thankgodr**
- Created a TODO List Project
- Setup Webpack
- Set up Views
- render view using webpack HTML plugin
- complete project requirement
- Lint code

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "This project is a simple HTML list of To Do tasks. The list will be styled according to the specifications listed below. This simple web page will be built using webpack and served by a webpack dev server.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack serve --open"
+  },
```
Comment (rukundoeric) [inline package.json]: - [x] <b>REQUIRED:</b> You are missing one important script that we need in order to be able to build our app. and that is `build`. Kindly add it.

File: src/js/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    const span = document.createElement('span');
+    span.className = 'fas fa-ellipsis-v pull-right';
+    listViewItem.appendChild(checkbox);
+    listViewItem.appendChild(document.createTextNode(task.description));
+    listViewItem.appendChild(span);
+    taskList.appendChild(listViewItem);
+  });
+}
+printInitialTasks();
```
Comment (rukundoeric) [inline src/js/index.js]: - [x] <b>REQUIRED:</b> The project require that there must be at least 3 default projects list on the page. Kindly fix that.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @thankgodr, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @thankgodr, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**